### PR TITLE
fix some bugs in demo.py, improve READEME.md

### PR DESCRIPTION
**Pull request by LyuDeepLearning**
fix some bug in demo.py, improve the path in READEME.md

### CODE REVIEW

File: scripts/demo.py [hunk 1]
```diff
@@ -41,7 +42,7 @@ def demo(config):
     image = Image.open(config.input_pic).convert('RGB')
     images = transform(image).unsqueeze(0).to(device)
 
-    model = get_model(args.model, pretrained=True, root=args.save_folder).to(device)
+    model = get_model(args.model,local_rank=args.local_rank,pretrained=True, root=args.save_folder).to(device)
```
Comment (Tramac) [inline scripts/demo.py:45]: `model = get_model(args.model, local_rank=args.local_rank, pretrained=True, root=args.save_folder).to(device)`

File: README.md [hunk 2]
```diff
@@ -56,7 +56,10 @@ python -m torch.distributed.launch --nproc_per_node=$NGPUS eval.py --model fcn32
 ### Demo
 ```
 cd ./scripts
-python demo.py --model fcn32s_vgg16_voc --input-pic ./datasets/test.jpg
+python demo.py --model fcn32s_vgg16_voc --input-pic ../datasets/test.jpg
+or
```
Comment (Tramac) [inline README.md:60]: Maybe the `or` should be removed.

**Review by Tramac — CHANGES_REQUESTED** [review]
Hello, there are some code style differences here, please fix it.

File: scripts/demo.py (hunk 1 above)
Comment (LyuDeepLearning) [reply scripts/demo.py:45]: okay, I got it!

File: README.md (hunk 2 above)
Comment (LyuDeepLearning) [reply README.md:60]: I add a new line below 'or':
python demo.py --model fcn32s_vgg16_voc --input-pic ../tests/test_img.jpg

**Review by Tramac — COMMENTED** [review]
Good job!

**Pull request merged**